qiskit_experiments/curve_analysis/base_curve_analysis.py

```diff
@@ -138,8 +138,12 @@ def _default_options(cls) -> Options:
             bounds (Dict[str, Tuple[float, float]]): Boundary of fit parameters.
                 The dictionary is keyed on the fit parameter names and
                 values are the tuples of (min, max) of each parameter.
+            fit_method (str): Fit method that LMFIT minimizer uses.
+                Default to ``least_squares`` method which implements the
+                Trust Region Reflective algorithm to solve the minimization problem.
+                See LMFIT documentation for available options.
             curve_fitter_options (Dict[str, Any]) Options that are passed to the
```

Comment:
Should this be renamed to `lmfit_options` now?

Reply:
Make sense.